query-engine/connector-test-kit/src/test/scala/util/TestDatabase.scala

```diff
@@ -19,7 +19,7 @@ case class MigrationEngine(project: Project) {
 
   def resetAndSetupDatabase(): Unit = {
     import scala.sys.process._
-    val cmd = List(EnvVars.migrationEngineBinaryPath, "cli", "-d", project.dataModelWithDataSourceConfig, "qe-setup")
+    val cmd = List(EnvVars.migrationEngineBinaryPath, "cli", "-d", project.envVar, "qe-setup")
```

Comment:
The name `.envVar` seems confusing in this context. How about we rename it to `dataModelWithDataSourceConfigBase64Encoded`?

Reply:
True.